explorer/src/Pos/Explorer/Txp/Local.hs

```diff
@@ -48,26 +48,31 @@ eTxProcessTransaction ::
        , CanJsonLog m
        )
     => Genesis.Config
+    -> TxValidationRules
     -> TxpConfiguration
     -> (TxId, TxAux)
     -> m (Either ToilVerFailure ())
-eTxProcessTransaction genesisConfig txpConfig itw =
+eTxProcessTransaction genesisConfig txValRules txpConfig itw =
     withStateLock LowPriority ProcessTransaction
-        $ \__tip -> eTxProcessTransactionNoLock genesisConfig txpConfig itw
+        $ \__tip -> eTxProcessTransactionNoLock genesisConfig txValRules txpConfig itw
 
+-- This returns a function to `txProcessTransactionAbstract` which then builds the TxValidationRules
+-- from the `configuration.yaml` and the state. Therefore we don't use `txValRules` here.
```

Comment:
Why do we pass a `TxValidationRules` if we never use it?

Reply:
Same Q

Reply:
See above comment.